.envrc

```diff
@@ -99,6 +113,17 @@ export NODE_OPTIONS=--max-old-space-size=4096
 # Enable this by default for development envs (CI/deploys do not use envrc)
 export SENTRY_UI_HOT_RELOAD=1
 
+## Initialize sentry-cli
+if ! require sentry-cli; then
+    curl -sL https://sentry.io/get-cli/ | bash
+fi
```

Comment:
Can we install only when we need to? i.e. in `report_to_sentry`

Reply:
That's fine with me.